Automattic/wp-calypso
# Marketplace: Add Education Footer Bottom Part

#### Changes proposed in this Pull Request
* Adds Education Footer Bottom Part

#### Testing Instructions
* Navigate to `http://calypso.localhost:3000/plugins/:siteId`
* Scroll down to the bottom
* Footer bottom part should appear with the right content.


#### Screenshots
| Mobile | Desktop |
|--------|-------|
|<img width="322" alt="Screen Shot 2022-03-15 at 11 31 05 PM" src="https://user-images.githubusercontent.com/1035546/158517116-85892a61-129b-4725-971c-de3b96c2d321.png">|<img width="1512" alt="Screen Shot 2022-03-15 at 11 30 49 PM" src="https://user-images.githubusercontent.com/1035546/158517156-5fc84394-7ab0-4d53-a6b3-f57571ffa61d.png">|



Related to #61491
